src/Api/Tracking.php

```diff
@@ -121,4 +122,31 @@ private static function get_logins(array $params): array
 
         return $response;
     }
+
+    /**
+     * Get last tracking data filtered by days.
+     *
+     * @param array $params refers to request params
+     * @return array Get tracking data.
+    */
+    private static function get_content_created(array $params): array
+    {
+        global $wpdb;
+
+        // phpcs:disable
+        $sql = 'SELECT count(ID) as total, post_type FROM %1$s WHERE post_date >= NOW() - INTERVAL %2$s DAY AND post_status = "publish" AND post_type IN ("post", "page", "p4_action") GROUP BY post_type';
+        $prepared_sql = $wpdb->prepare($sql, $wpdb->posts, $params['last_days']);
```

Comment:
Using a table name as a parameter in a prepare() query is highly discouraged. MySQL does not support parameterizing table names in prepared statements, which can lead to SQL errors.
More details: [Stack Overflow - Can I parameterize the table name in a prepared statement?](https://stackoverflow.com/questions/11312737/can-i-parameterize-the-table-name-in-a-prepared-statement)